internal/log/mode/type.go

```diff
@@ -14,22 +14,22 @@
 // limitations under the License.
 //
 
-package loggertype
+package mode
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
Why did you rename this package?
The previous one is the better naming in this context, because it represents exactly what it is.
"mode" has ambiguous meaning in contexts of software designing, so think it's not a good choice to use "mode" for package name.
Other suggestion, "kind", "implementation", or "logger_impl", etc ...

Reply:
loggerType is not suitable naming for golang's package we should name it as logger/type but we can't make type package and log/logger/type is boring name
I think we should use another naming that's why I changed package name as single word mode

Reply:
I understand your suggestion, but I think mode is ok

Reply:
kind is something like different categories (eg. Deployment, Service, Ingress) , or we can use method as naming what do you think

Reply:
hmm. i see.
just "logger" is the best naming for this package i think, but the word "logger" is used in many packages in this repository. We'll have a lot of work to fix them. :thinking:
also,  `internal/cache/cacher` may be the better naming for `internal/cache/mode`.

Reply:
hmm, okay sounds great I'll fixit soon

Reply:
I'll make cacher package and `type Type uint8` in it

Reply:
fixed